theano/gof/cmodule.py

```diff
@@ -1679,15 +1681,19 @@ def std_lib_dirs_and_libs():
         # get the name of the python library (shared object)
         libname = distutils.sysconfig.get_config_var("LDLIBRARY")
 
-        if libname.startswith("lib"):
-            libname = libname[3:]
+        if libname:
+            if libname.startswith("lib"):
+                libname = libname[3:]
 
-        # remove extension if present
-        if libname.endswith(".so"):
-            libname = libname[:-3]
-        elif libname.endswith(".a"):
-            libname = libname[:-2]
+            # remove extension if present
+            if libname.endswith(".so"):
+                libname = libname[:-3]
+            elif libname.endswith(".a"):
+                libname = libname[:-2]
+        else:
+            libname = "pypy-c"
 
+        # Must create a copy of pypy-c in prefix/lib
```

Comment:
I do not understand this comment. Can you explain more? Is that a modification the user must do himself or this is already done automatically?

Reply:
To originally minimize the changes, I just symlinked the .so into a manually created lib folder in the right location. From looking around, it seems like the only out-of-the-box compatible solution at the moment would be to search for the .so in the sys.executable (bin) directory if we detect we are running PyPy.